ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -159,3 +159,10 @@ def tanh(self, name=None):
     @with_supported_dtypes({"2.4.3 and above": ("float32", "float64")}, "paddle")
     def add_(self, name=None):
         return ivy.add(self._ivy_array)
+
+    with_supported_dtypes(
+        {"2.4.2 and below": ("float16", "float32", "float64")}, "paddle"

```

Comment:
Hey! seems like `float16` is actually unsupported here, would you mind to remove it from the list? Thanks!

Reply:
I checked paddle docs and it says it can take float16 as input.

Reply:
yeah, but that might be for the functional only as I tried the method out and it gave me errors

Reply:
Ok I am going to change it thanks 👍

Reply:
I have removed float16.